File: mk1/Lista3/Zad2/Prostokat.cpp
#include "Prostokat.hpp"

Prostokat::Prostokat(int a, int b, int c, int d, int kat) noexcept(false) {
    bok.push_back(a);
    bok.push_back(b);
    bok.push_back(c);
    bok.push_back(d);
    this->kat = kat;
}

std::string Prostokat::Nazwa() noexcept(true) {
    return "Prostokat";
}

File: mk1/mk1/Lista3/Zad1/Prostokat.java
class Prostokat extends Czworokat{
    
    Prostokat(float a, float b, float c, float d, float alf){
        this.bok1 = a;
        this.bok2 = b;
        this.bok3 = c;
        this.bok4 = d;
        this.kat = alf;
    }

    public float Pole(){
        return bok1*bok2;
    }

    public float Obwod(){
        return 2*bok1+2*bok2;
    }

    public String Nazwa(){
        return "Prostokat";
    }


}